### DISCUSSION TITLE
Do we need to upload AAB file (Android) to Release assets?

### DISCUSSION THREAD

**Discussion by Districh-ru** [ihhub/fheroes2#7307] [General]
As I understand, AAB file cannot be directly installed on Android devices. It is used only by Google Play and other markets.
To install it the user must use some software (e.g. https://github.com/google/bundletool ) to build APK file.
But we already have the APK file in Android archive.
Is there a need to put ABB file into the Android archive for Release assets?

**Comment by Districh-ru** [comment]
Hello, @oleg-derevenetz , what is your opinion about need of AAB in Release assets?

**Reply by oleg-derevenetz** [reply]
Hi @Districh-ru well, formally, it is not needed there. But this means that the corresponding ZIP file should be re-packed by @ihhub manually (because originally AAB is here to put it to the Google Play).

**Reply by ihhub** [reply]
Then I shall remove AAB from the release archive.

**Reply by Districh-ru** [reply]
> Hi @Districh-ru well, formally, it is not needed there. But this means that the corresponding ZIP file should be re-packed by @ihhub manually (because originally AAB is here to put it to the Google Play). 

Is it possible to make a separate ZIP archive for AAB file in Github workflow for Android?

**Reply by oleg-derevenetz** [reply]
Yes, we can make two separate archives. I'll do this later today.

**Reply by Districh-ru** [reply]
> Yes, we can make two separate archives. I'll do this later today.

Great, thanks! :)
